youtube_dl/extractor/hanime.py

```diff
@@ -0,0 +1,101 @@
+        },
+    }
+
+    def _real_extract(self, url):
+        video_slug = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_slug)
+        page_json = self._html_search_regex(r'window.__NUXT__=(.+?);<\/script>', webpage, 'Inline JSON')
+        page_json = self._parse_json(page_json, video_slug).get('state').get('data').get('video').get('hentai_video')
+        api_json = self._download_json(
+            'https://members.hanime.tv/api/v3/videos_manifests/%s' % video_slug,
+            video_slug,
+            'API Call', headers={'X-Directive': 'api'}).get('videos_manifest').get('servers')[0].get('streams')
+
+        title = page_json.get('name') or api_json.get[0].get('video_stream_group_id')
+        tags = [t.get('text') for t in page_json.get('hentai_tags')]
+
+        formats = []
+        for f in api_json:
+            item_url = url_or_none(f.get('url')) or url_or_none('https://hanime.tv/api/v1/m3u8s/%s.m3u8' % f.get('id'))
+            format = [{
+                'width': int_or_none(f.get('width')),
+                'height': int_or_none(f.get('height')),
+                'filesize_approx': parse_filesize('%sMb' % f.get('filesize_mbs')),
+                'protocol': 'm3u8',
+                'format_id': 'mp4-%sp' % f.get('height'),
+                'tbr': float_or_none(float_or_none(f.get('filesize_mbs'), invscale=8388), int_or_none(f.get('duration_in_ms'), 1000)),
+                'ext': 'mp4',
+                'url': item_url,
+            }, {
+                'width': int_or_none(f.get('width')),
+                'height': int_or_none(f.get('height')),
+                'protocol': 'https',
+                'format_id': 'm3u8-%sp' % f.get('height'),
+                'format_note': '~8-50.00Kib',
+                'ext': 'm3u8',
+                'url': item_url,
+            }]
+            for i in format:
+                formats.append(i)
+        formats.reverse()
+
+        return {
+            'id': str_or_none(api_json[0].get('id')),
+            'display_id': video_slug,
+            'title': title,
+            'description': clean_html(page_json.get('description')).strip(),
+            'thumbnails': [
+                {'preference': 0, 'id': 'Poster', 'url': url_or_none(page_json.get('poster_url'))},
+                {'preference': 1, 'id': 'Cover', 'url': url_or_none(page_json.get('cover_url'))},
+            ],
+            'release_date': unified_strdate(page_json.get('released_at')),
+            'upload_date': unified_strdate(page_json.get('created_at')),
+            'creator': str_or_none(page_json.get('brand')),
+            'view_count': int_or_none(page_json.get('views')),
+            'like_count': int_or_none(page_json.get('likes')),
+            'dislike_count': int_or_none(page_json.get('dislikes')),
+            'duration': parse_duration('%sms' % f.get('duration_in_ms')),
+            'tags': tags,
+            'censored': str_or_none(page_json.get('is_censored')),
```

Comment:
No such meta field.

Reply:
Is there any way I can pass this information to use with the output layout?

Reply:
Depends on what does it mean.

Reply:
Never mind, it is also a tag, which can't be used there either so makes there's no reason this one should.